Dockerfile

```diff
@@ -1,19 +1,28 @@
-FROM ubuntu:bionic
-MAINTAINER Zach Halpern <zahalpern+github@gmail.com>
+FROM ubuntu:noble
```

Comment:
Can you use the numbered version over the name, please?

Reply:
Makes sense & done